services/src/main/scala/cromwell/pipeline/service/VersioningException.scala

```diff
@@ -0,0 +1,11 @@
+package cromwell.pipeline.service
+
+sealed abstract class VersioningException extends Exception
```

Comment:
Maybe it could be useful to make ```message``` a field of ```VersioningException```?

Reply:
It is done